shell/packages/sandstorm-ui-app-details/app-details-client.js

```diff
@@ -32,7 +32,7 @@ var latestAppManifestForAppId = function (db, appId) {
 
 var getAppTitle = function (appDetailsHandle) {
   var pkg = latestPackageForAppId(appDetailsHandle._db, appDetailsHandle._appId);
-  return SandstormDb.appNameFromPackage(pkg);
+  return pkg && SandstormDb.appNameFromPackage(pkg) || "<unknown>";
```

Comment:
The `unknown` case should never happen, right? If that's accurate, then probably we should (in the future, not necessarily now) do something to log the details of this case because if we ever do return `<unknown>` it's because of a bug somewhere.